# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-hal/src/gles/shader_clear.frag
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/wgpu-hal/src/gles/shader_clear.frag b/third_party/rust/wgpu-hal/src/gles/shader_clear.frag
new file mode 100644
--- /dev/null
+++ b/third_party/rust/wgpu-hal/src/gles/shader_clear.frag
@@ -0,0 +1,7 @@
+#version 300 es
+precision lowp float;
+uniform vec4 color;
+out vec4 frag;
+void main() {
+  frag = color;
+}
\ No newline at end of file